src/toil/test/jobStores/jobStoreTest.py

```diff
@@ -1237,23 +1240,21 @@ def testSDBDomainsDeletedOnFailedJobstoreBucketCreation(self):
         different region with the same name.
         """
         from boto.sdb import connect_to_region
-        from boto.s3.connection import Location, S3Connection
-        from boto.exception import S3ResponseError
         from toil.jobStores.aws.jobStore import BucketLocationConflictException
-        from toil.jobStores.aws.utils import retry_s3
-        externalAWSLocation = Location.USWest
+        from toil.jobStores.aws.jobStore import get_boto3_session
+        externalAWSLocation = 'us-west-1'
```

Comment:
Nit: This could probably be set in the main toil test config and imported to ensure all of our tests are in the same region.

Reply:
I'm not entirely sure what this is referring to.